src/Gateway/Card.php

```diff
@@ -1,8 +1,6 @@
 <?php
 
-namespace Laravel\Cashier;
-
-use Stripe\Card as StripeCard;
+namespace Laravel\Cashier\Gateway;
 
 class Card
```

Comment:
It may be possible to bring some of the Stripe-only functionality, like card management, into Braintree.